docs/en/observability/monitor-infra/host-details-partial.asciidoc

```diff
@@ -0,0 +1,149 @@
+`idle`, `zombie`, and `unknown`.
+
+|=== 
+====
+
+[%collapsible]
+.*Logs*
+====
+
+[role="screenshot"]
+image::images/logs-overlay.png[Host logs]
+
+The *Logs* tab displays logs relating to the host that you have selected. By default, the logs tab displays the following columns. 
+
+|=== 
+
+| *Timestamp* | The timestamp of the log entry from the `timestamp` field. 
+
+| *Message* | The message extracted from the document.
+The content of this field depends on the type of log message.
+If no special log message type is detected, the {ecs-ref}/ecs-base.html[Elastic Common Schema (ECS)]
+base field, `message`, is used.
+
+|=== 
+
+You can customize the logs view by adding a column for an arbitrary field you would like
+to filter by. For more information, refer to <<customize-stream-page,Customize Stream>>.
+To view the logs in the {logs-app} for a detailed analysis, click *Open in Logs*.
+====
+
+[%collapsible]
+.*Anomalies*
+====
+
+[role="screenshot"]
+image::images/anomalies-overlay.png[Anomalies]
+
+The *Anomalies* table displays a list of each single metric {anomaly-detect} job for the specific host. By default, anomaly
+jobs are sorted by time to show the most recent job. 
+
+Along with the name of each anomaly job, detected anomalies with a severity score equal to 50, or higher, are listed. These
+scores represent a severity of "warning" or higher in the selected time period. The *summary* value represents the increase between
+the actual value and the expected ("typical") value of the host metric in the anomaly record result.
+
+To drill down and analyze the metric anomaly, select *Actions -> Open in Anomaly Explorer* to view the
+{ml-docs}/ml-gs-results.html[Anomaly Explorer in {ml-app}]. You can also select *Actions -> Show in Inventory* to view the host
+Inventory page, filtered by the specific metric. 
+====
+
+[%collapsible]
+.*Osquery*
+====
+
+[IMPORTANT]
+=====
+You must have an active {fleet-guide}/elastic-agent-installation.html[{agent}] with an assigned agent policy
+that includes the {integrations-docs}/osquery_manager.html[Osquery Manager]
+integration and have Osquery {kibana-ref}/kibana-privileges.html[{kib} privileges] as a user.
+=====
+
+[role="screenshot"]
+image::images/osquery-overlay.png[Osquery]
+
+The *Osquery* tab allows you to build SQL statements to query your host data.
+You can create and run live or saved queries against
+the {agent}. Osquery results are stored in {es}
+so that you can use the {stack} to search, analyze, and
+visualize your host metrics. To create saved queries and add scheduled query groups,
+refer to {kibana-ref}/osquery.html[Osquery].
+
+In the example above, we query for the top 5 memory hogs running on the host.
+Under the *Results* tab, the total virtual memory size (`total_size` renamed to
+`memory_used` to be a little more user friendly) is returned in descending order,
```

Comment:
It looks like the new screenshot doesn't show the **Results** or **Status** tab, so the user won't see it in the example. Not sure if this would maybe be more meaningful showing those tabs, if the tabs still exist?

Reply:
@mdbirnstiehl Good catch! I should have checked the new screen to make sure it had the query details. Unfortunately this screen requires a setup I don't currently have. When I enter the query shown in the screen capture and click Submit, it doesn't do anything (probably because I haven't installed the agent).

I also think it's not great that we refer to an example query that _only_ appears in the screen capture (not great for accessibility).

To get this merged today, I am going to remove the content that I can't validate and follow up with Roshan tomorrow about getting an updated screen shot. Thanks!

Reply:
I'm going to leave the bit about the tabs because I'm pretty sure they still exist, but I will confirm when I see the updated screenshot.

Opened an issue to make sure we get that screenshot updated: https://github.com/elastic/observability-docs/issues/3350